.rubocop.yml

```diff
@@ -51,7 +51,8 @@ Layout/FirstParameterIndentation:
 
 # Use Ruby >= 1.9 syntax for hashes. Prefer { a: :b } over { :a => :b }.
 Style/HashSyntax:
-  Enabled: false
+  Enabled: true
+  EnforcedStyle: ruby19
```

Comment:
This is default, do we need to explicitly state it?

Reply:
It's the style in the config file so far so I would just leave it like this.